ci_pod/jupyterlab/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM fedora:38
+
+ARG JUPYTERLAB_VERSION
+ENV JUPYTERLAB_VERSION=$JUPYTERLAB_VERSION
+ARG MAINTAINER
+ENV MAINTAINER=$MAINTAINER
+
+LABEL maintainer="Your Name <$MAINTAINER>"
+
+# Install system dependencies
+RUN dnf update -y && \
```

Comment:
Don't do the `dnf -y update`; it will add possibly quite a lot of size to the image.

Reply:
That's certainly a valid reason to do so.